src/HealthChecks.ClickHouse/README.md

```diff
@@ -0,0 +1,39 @@
+## ClickHouse Health Check
+
+This health check verifies the ability to communicate with [ClickHouse](https://www.clickhouse.com/). It uses the [ClickHouse.Client](https://www.nuget.org/packages/ClickHouse.Client) library.
+
+## Recommended approach
+
+When registering the ClickHouse health check, it is recommended to use a shared `HttpClient` instance so that you can take advantage of the built-in [pooling](https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines#pooled-connections).
+
+We encourage you to use [ClickHouse.Extensions.DependencyInjection](https://www.nuget.org/packages/ClickHouse.Extensions.DependencyInjection) package. It allows easy configuration of your ClickHouse connections and registers the appropriate services in your DI container.
+
+```csharp
+void Configure(IServiceCollection services)
+{
+    services.AddClickHouseDataSource("Host=ch;Username=default;Password=test;Database=default");
```

Comment:
since https://github.com/DarkWanderer/ClickHouse.Client/pull/553 has not been merged  yet, I assume that `AddClickHouseDataSource` does not exist as of today?